DESCRIPTION

```diff
@@ -1,8 +1,8 @@
 Package: BASiCS
 Type: Package
 Title: Bayesian Analysis of Single-Cell Sequencing data
-Version: 2.1.7
-Date: 2020-09-01
+Version: 2.2.0
```

Comment:
We do not change y in the version number. That's done by the BioC team once the new release is up. 

Version number should be 2.1.8
Date should be 2020-09-23